### PR TITLE
Added support for latest exporter

### PR DESCRIPTION
**Pull request by core23**
## Subject

I am targeting this branch, because this is BC.

## Changelog

```markdown
### Added
- Added support for latest exporter
```

### CODE REVIEW

File: src/Sitemap/SourceManager.php [hunk 1]
```diff
@@ -13,8 +13,8 @@
 
 namespace Sonata\SeoBundle\Sitemap;
 
-use Exporter\Source\ChainSourceIterator;
-use Exporter\Source\SourceIteratorInterface;
+use Sonata\Exporter\Source\ChainSourceIterator;
+use Sonata\Exporter\Source\SourceIteratorInterface;
```
Comment (greg0ire) [inline src/Sitemap/SourceManager.php]: Please trigger autoload for that one (it is used in `addSource`'s signature)

Comment (core23) [reply src/Sitemap/SourceManager.php]: I'm not sure if this is the right solution...

You didn't add the trigger neither
https://github.com/sonata-project/SonataAdminBundle/pull/5408

Comment (greg0ire) [reply src/Sitemap/SourceManager.php]: In that PR, the type hint appear in a constructor, and constructors don't have to be compatible with the constructor of the parent class. If you want to be sure this is the right solution, let me provide you with a test.

Comment (greg0ire) [reply src/Sitemap/SourceManager.php]:
```
use Exporter\Source\SourceIteratorInterface;
use Sonata\SeoBundle\Sitemap\SourceManager;

final class ExtendingSourceManager extends SourceManager
{
    public function addSource($group, SourceIteratorInterface $source, array $types = [])
    {
    }
}
```
You can test that this crashes, and that it no longer crashes with the `class_exists`.

Comment (greg0ire) [reply src/Sitemap/SourceManager.php]: Well… I feel dumb now, this does not crash.

Comment (greg0ire) [reply src/Sitemap/SourceManager.php]: Ah for that one I know, `SourceManager` implements `SourceManagerInterface`, and `implements` triggers autoloading :) . That one makes sense 👍

**Comment by greg0ire** [comment]
Thanks @core23 !

**Pull request merged**